### PR TITLE
Bugfix for enforce module use policy. 

### PR DESCRIPTION
**Pull request by cdhesse**

### CODE REVIEW

File: plan/enforce-module-use-policy.rego [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package spacelift
 
+import future.keywords
```
Comment (jmfontaine) [inline plan/enforce-module-use-policy.rego:3]: Isn't this enough?

```suggestion
import future.keywords.in
```

**Review by jmfontaine — COMMENTED** [review]
Aside from the minor import suggestion and the failing formatting check, it looks good to me. Thanks, @cdhesse!

**Review by jmfontaine — CHANGES_REQUESTED** [review]
Thanks for the change.

The last blocker is the failing code formatting check. You can fix that by running `opa fmt -w plan/enforce-module-use-policy.rego` at the root of the repository and committing the changes.

**Comment by cdhesse** [comment]
> opa fmt -w plan/enforce-module-use-policy.rego

I've tried to get opa to work, but i keep getting errors.

failed to parse Rego source file: 1 error occurred: plan/enforce-module-use-policy.rego:3: rego_parse_error: invalid path future.keywords: path must begin with input or data
        import future.keywords

Even removing the import (which of course I need) - i get other errors.  I can tell you this policy works in our environment (the initial commit does not work however).

**Review by jmfontaine — APPROVED** [review]
@cdhesse Okay, I will merge it as it is, and I will figure it out later. Thanks for your contribution!

**Comment by jmfontaine** [comment]
Thanks @cdhesse!

**Pull request merged**